docs/ruleset_functions/get_zone_conditioning_category.md

```diff
@@ -64,7 +66,7 @@ Logic:
       - For each surface in zone: `for surface in zone.surfaces:`  
 
         - Check if surface is interior, get adjacent zone: `if surface.adjacent_to == "INTERIOR": adjacent_zone = match_data_element(RMR, zones, surface.adjacent_zone_id)`  
-
+**[CH: There is no `surface.fenestration_subsurfaces` field. There is a `surface.subsurfaces` field instead. Subsurface has `opaque_area` and `glazed_area`.]**
```

Comment:
Surface.fenestration_subsurfaces is no longer in the schema. I'll correct the calculation below.

Reply:
This is corrected.